index.html

```diff
@@ -10,9 +10,9 @@
     <H1>Forms</H1>
     <form action="https://sandbox.merge.academy/api/message" method="POST">
       <input required placeholder="name" name="name" type="text">
-      <br>
+       <br>
       <label for="message"></label>
-      <br>
+       <br>
       <textarea
        required placeholder="Your message"
```

Comment:
```suggestion
       required
       placeholder="Your message"
```
якщо автовирівнювач не досить коректно вирівнює, роби це власноруч